Note SBUS compatibility is broader than previously listed (#3412)
PX4/PX4-user_guide: en/getting_started/rc_transmitter_receiver.md

It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware.

::: info
This remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) or [TBS Crossfire](../telemetry/crsf_telemetry.md) radio modules.
:::

Other popular transmitter/receiver pairs

- Turnigy remote using, for example, the FrSky transmitter/receiver modules.
- Futaba Transmitters and compatible Futaba S-Bus receivers.
- Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
- Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)

### PX4-Compatible Receivers {#compatible_receivers}

In addition to the transmitter/receiver pairs being compatible, the receiver must also be compatible with PX4 and the flight controller hardware.

_PX4_ and _Pixhawk_ have been validated with:

- PPM sum receivers
- S.BUS and S.BUS2 receivers from
  - Futaba
  - FrSky S.BUS and PPM models
  - TBS Crossfire with SBUS as output protocol
  - Herelink

> - TBS Crossfire with (CRSF protocol)
> - Express LRS with (CRSF protocol)

Author: ```suggestion
- TBS Crossfire with ([CRSF protocol](../telemetry/crsf_telemetry.md))
- Express LRS with ([CRSF protocol](../telemetry/crsf_telemetry.md))

```

::: info
This remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) or [TBS Crossfire](../telemetry/crsf_telemetry.md) radio modules.
:::

Other popular transmitter/receiver pairs

- Turnigy remote using, for example, the FrSky transmitter/receiver modules.
- Futaba Transmitters and compatible Futaba S-Bus receivers.
- Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
- Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)

### PX4-Compatible Receivers {#compatible_receivers}

In addition to the transmitter/receiver pairs being compatible, the receiver must also be compatible with PX4 and the flight controller hardware.

_PX4_ and _Pixhawk_ have been validated with:

- PPM sum receivers
- S.BUS and S.BUS2 receivers from
  - Futaba
  - FrSky S.BUS and PPM models
  - TBS Crossfire with SBUS as output protocol
  - Herelink
- TBS Crossfire with ([CRSF protocol](../telemetry/crsf_telemetry.md))
- Express LRS with ([CRSF protocol](../telemetry/crsf_telemetry.md))

- TBS Ghost with (GHST protocol)
- Spektrum DSM
- Graupner HoTT

> Receivers from other vendors using one of the supported protocols are likely to work but have not been tested.

Author: ```suggestion
Receivers from other vendors that use a supported protocol are likely to work but have not been tested.

```

One of the most popular RC units is the _FrSky Taranis X9D_.
It has an internal transmitter module can be used with the recommended _FrSky X4R-SB_ (S-BUS, low delay) or _X4R_ (PPM-Sum, legacy) receivers out of the box.
It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware.

::: info
This remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) or [TBS Crossfire](../telemetry/crsf_telemetry.md) radio modules.
:::

Other popular transmitter/receiver pairs

- Turnigy remote using, for example, the FrSky transmitter/receiver modules.
- Futaba Transmitters and compatible Futaba S-Bus receivers.
- Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
- Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)

### PX4-Compatible Receivers {#compatible_receivers}

In addition to the transmitter/receiver pairs being compatible, the receiver must also be compatible with PX4 and the flight controller hardware.

_PX4_ and _Pixhawk_ have been validated with:

- PPM sum receivers

> - S.BUS and S.BUS2 receivers from

Author: ```suggestion
- S.BUS and S.BUS2 receivers from:

```

  - Futaba
  - FrSky S.BUS and PPM models
  - TBS Crossfire with SBUS as output protocol
  - Herelink
- TBS Crossfire with ([CRSF protocol](../telemetry/crsf_telemetry.md))
- Express LRS with ([CRSF protocol](../telemetry/crsf_telemetry.md))

- TBS Ghost with (GHST protocol)
- Spektrum DSM
- Graupner HoTT

Receivers from other vendors that use a supported protocol are likely to work but have not been tested.

>   ::: info
>   Differences between receiver models and no detailed specification of protocols historically resulted in incompatibilities.
>   The receivers we have tested all now appear to be compatible, but it is possible that others may not be.
>   :::

Author: ```suggestion
::: info
Historically there were differences and incompatibilities between receiver models, largely due to a lack of detailed specification of protocols.
The receivers we have tested all now appear to be compatible, but it is possible that others may not be.
:::
```

It has an internal transmitter module can be used with the recommended _FrSky X4R-SB_ (S-BUS, low delay) or _X4R_ (PPM-Sum, legacy) receivers out of the box.
It also has a custom radio transmitter module slot and customizable open source OpenTX Firmware.

::: info
This remote control unit can display vehicle telemetry when used with [FrSky](../peripherals/frsky_telemetry.md) or [TBS Crossfire](../telemetry/crsf_telemetry.md) radio modules.
:::

Other popular transmitter/receiver pairs

- Turnigy remote using, for example, the FrSky transmitter/receiver modules.
- Futaba Transmitters and compatible Futaba S-Bus receivers.
- Long range ~900MHz, low latency: "Team Black Sheep Crossfire" or "Crossfire Micro" set with a compatible remote (e.g. Taranis)
- Long Range ~433MHz: ImmersionRC EzUHF set with a compatible remote (e.g. Taranis)

### PX4-Compatible Receivers {#compatible_receivers}

In addition to the transmitter/receiver pairs being compatible, the receiver must also be compatible with PX4 and the flight controller hardware.

_PX4_ and _Pixhawk_ have been validated with:

- All Spektrum DSM RC receivers
- All Futaba S.BUS and S.BUS2 RC receivers.

> - All FrSky S.BUS and PPM models

Author: Would this mean S.BUS and S.BUS2 (i.e. is the situation the same for both)?

Reviewer: I think SBUS 1 vs. 2 is a distinction from the transition time in 2013. We might mention that it is backward compatible in one place, but I expect most people are not explicitly looking to fit 10+ year old devices.

- S.BUS and S.BUS2 RC receivers from Futaba, FrSky, TBS Crossfire with SBUS as output protocol, Herelink, Futaba SBUS remote.
  Receivers from other vendors are likely to work but have not been tested.

  ::: info
  There is no detailed specification of the protocol, which historically resulted in slight incompatibilities.
  The receivers we have tested all now appear to be compatible, but it is possible that others may not be.
  :::

- All PPM models (from any manufacturer)
- Graupner HoTT

> - TBS Crossfire/Express LRS Receivers using [CRSF Telemetry](../telemetry/crsf_telemetry.md) (UART connection).

Author: @MaEtUgR This is the only section that had an actual content change. Can you please check it.

Reviewer: @hamishwillee Thank you for taking this up and improving the documentation.
Form my perspective there are two main problems with this formulation.
- I'd be careful saying "All" for each of the brands because what happens is we flew a bunch of models but there are new ones coming out each year and while they are most likely to be compatible we cannot state we tested them because I'm not aware someone buys each model and tests it.
- There's duplication e.g.
   - All Futaba S.BUS and S.BUS2 RC receivers
   - ...
   - S.BUS and S.BUS2 RC receivers from Futaba, ...

Maybe I'll just add another commit with some suggestions.

Author: @MaEtUgR Yes, I saw the duplication - it came from the fact that previously we supported each protocol but only on some devices. Given we can't say "All" and also "we tested this" then perhaps we should say something like (?)

----

_PX4_ and _Pixhawk_ are expected to work with:

- SBUS Receivers
- PPM Receivers
- Spectrum DSM RC Receivers 
- Graupner HoTT
- TBS Crossfire/Express LRS Receivers using [CRSF Telemetry](https://github.com/PX4/PX4-user_guide/telemetry/crsf_telemetry.md) (UART connection).

> **Note:** If testing shows a receiver of these types that doesn't work then raise an issue.
